src/cmd-remote-build

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python3 -u
+
+import sys
+import argparse
+from os import environ
+from subprocess import check_call, check_output, CalledProcessError
+
+def run_cmd(command, output=None):
+    '''
+    Run the given command using check_call and verify its return code.
+    @param str command command to be executed
+    '''
+    if output:
+        try:
+            output = check_output(command.split())
+        except CalledProcessError as e:
+            print('An exception has occurred: {0}'.format(e))
+            sys.exit(1)
```

Comment:
Minor nit, I think this is more concise as `raise SystemExit(f"Failed to invoke command: {e}")` or so.